### ISSUE TITLE
plots for visualizing subpopulation wastewater concentrations and Rt

### ISSUE THREAD

**Issue by sbidari** [CDCgov/pyrenew-hew#61]
We want to create plots of subpopulation specific posterior/predictive distributions. 
- [ ]   titles/labels to indicate which subpop is being plotted
- [ ]   overlay site/subpop data over predictions when available

**Comment by dylanhmorris** [comment]
We have these for wastewater but not for R(t), correct @sbidari?

**Comment by sbidari** [comment]
Correct! Although this PR was for the wastewater model in `demos/ww_model`

**Comment by damonbayer** [comment]
We are planning to start over wrt wastewater modeling when we switch to the revised pyrenew-multisignal.
